src/domain/agent/credential/CredentialCard.tsx

```
@@ -62,14 +54,14 @@ let i = 0;
 const claimParser = claims => {
   return (
     <>
-      <WrapperTypography variant="h6" color="neutralMedium">
+      <Typography variant="h6" color="neutralMedium.main" fontWeight="medium" fontSize={16}>
         Claims:
-      </WrapperTypography>
+      </Typography>
       {claims.map(claim => (
         <Box sx={{ whiteSpace: 'initial', wordBreak: 'break-all' }}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)